hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -97,11 +99,7 @@ public static TypedProperties loadGlobalProps() {
     if (defaultConfPath.isPresent()) {
       conf.addPropsFromFile(defaultConfPath.get());
     } else {
-      try {
-        conf.addPropsFromFile(new Path(DEFAULT_CONF_FILE_DIR));
-      } catch (Exception ignored) {
```

Comment:
this is actually preferrable to leaking the check for a default path inside `addPropsFromFile`

Reply:
Comments addressed in #5311 .